randovania/games/samus_returns/logic_database/Area 4 Central Caves.txt

```diff
@@ -696,12 +696,17 @@ Extra - asset_id: collision_camera_013
   * Pickup 85; Category? Minor
```

Comment:
I assume A4 **Crystal Mines** :
- Green crystal dugout/mines intersection terminal is too high?
- gemstone gorge / pink crystal preserve is too weird with the slope?

Reply:
Both are too high